Cargo.toml

```diff
@@ -158,7 +352,7 @@ runtime-common = { path = "runtime/common" }
 frame-benchmarking = { git = "https://github.com/paritytech/substrate", optional = true, default-features = false, branch = "polkadot-v0.9.43" }
 frame-system-benchmarking = { git = "https://github.com/paritytech/substrate", optional = true, default-features = false, branch = "polkadot-v0.9.43" }
 # integration testing
-runtime-integration-tests = { path = "runtime/integration-tests", optional = true, default-features = false }
+#runtime-integration-tests = { path = "runtime/integration-tests", optional = true, default-features = false }
```

Comment:
???

Reply:
Why were we linking integration tests as a dependency of our node before?

Reply:
I thought we needed it for some features. Fine then